Tasks/Gradle/task.json

```diff
@@ -204,6 +214,7 @@
         },
         {
             "name": "gradleOpts",
+            "aliases": [ "gradleOptions" ],
```

Comment:
Options is a bit confusing here because it isn't the "Option" suffix. Not sure it matters, just noticed it.

Reply:
I know what you mean.  My thinking is 'options' (plural) for command line args to a tool if 'Options' is the label in the UI.  'Option' singular for a choice in a pickList.